netmaster/docknet/docknet_test.go

```diff
@@ -225,3 +224,21 @@ func TestDocknetCreateDelete(t *testing.T) {
 	checkDocknetCreate(t, "unit-test", "net1", "srv1", "10.1.1.1/24", "10.1.1.254")
 	checkDocknetDelete(t, "unit-test", "net1", "srv1")
 }
+
+func TestUpdatePluginName(t *testing.T) {
+	expectNetDriver := "bridge"
+	expectIPAMDriver := "default"
+	UpdatePluginName("bridge", "default")
+
+	if expectNetDriver != netDriverName {
```

Comment:
how is netDriverName in scope?

Reply:
Since this Go test file is in the `docknet` package (see top of file), everything in the `docknet` package is available here.

This var is defined in docknet.go here:  https://github.com/kahou82/netplugin/blob/37a3b829c00226b8af742f80e6c46889c20fd4d2/netmaster/docknet/docknet.go#L44

Reply:
👍